support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/PrintProductsHeading.tsx

```diff
@@ -0,0 +1,94 @@
+import { css } from '@emotion/react';
+import type { ActiveRatePlanKey } from 'helpers/productCatalog';
+
+const yellowHighlightText = css`
+	background-color: #ffe500;
+	padding: 0 5px;
+`;
+
+function YellowHighlightText({ text }: { text: string }) {
+	return <span css={yellowHighlightText}>{text}</span>;
+}
+
+export default function PrintProductsHeading({
+	isObserverPrint,
```

Comment:
Do wonder if you can tell your an `isObserverPrint` by passing in `ActiveProductKey` & `ActiveRatePlanKey` and working it out later down the react tree. Probably applies to a few components. Don't suppose much in it however?